docs/faq.rst

```diff
@@ -76,7 +76,7 @@ Can I use KSQL with my favorite data format (e.g. JSON, Avro)?
 
 KSQL currently supports formats:
 
--  DELIMITED (e.g. CSV)
+-  DELIMITED (e.g. comma-separated value)
```

Comment:
Might be better to use the acronym and the long form, so "CSV, ie. comma separated value".